### ISSUE TITLE
`get_mean_pairwise_distance` hangs (>10 min) on deserialized tree

### ISSUE THREAD

**Issue by mmore500** [emilydolson/python-phylogeny-tracker#87]
**Describe the bug**
`get_mean_pairwise_distance` hangs (>10 min) and cannot be interrupted for valid alife standard file.

**To Reproduce**
```python3
from phylotrackpy import systematics
syst = systematics.Systematics(lambda x: x)
syst.load_from_file("test.csv", "id", True)
syst.get_mean_pairwise_distance(True)
```
[test.csv](https://github.com/emilydolson/phylotrackpy/files/14748209/test.csv)

**Expected behavior**
Should not hang.

**Screenshots**
![tree (2)](https://github.com/emilydolson/phylotrackpy/assets/10763333/2332e86a-8edb-4ddd-8500-0e4b033a219b)

**Computational environment (please complete the following information):**
 - OS: fedora
 - Python version: 3.10.13
 - phylotrackpy v0.2.0

**Comment by mmore500** [comment]
Issue persists even if origin time -1 inside of csv is changed to 0.

**Comment by mmore500** [comment]
Seems potentially related to absence of `destruction_time` column.

**Comment by mmore500** [comment]
Setting `destruction_time` equal to `origin_time` causes hang
[pq-f5320699-6e91-4f5d-babe-81d7169e627c.csv](https://github.com/emilydolson/phylotrackpy/files/14748601/pq-f5320699-6e91-4f5d-babe-81d7169e627c.csv)

**Comment by mmore500** [comment]
However, setting `nan` destruction times to 100000 does not hang.
[gq-529b3c65-6d1c-4922-bd64-c95c9aead237.csv](https://github.com/emilydolson/phylotrackpy/files/14748615/gq-529b3c65-6d1c-4922-bd64-c95c9aead237.csv)

**Comment by mmore500** [comment]
note: `get_mean_pairwise_distance` is also missing API documentation

**Comment by emilydolson** [comment]
Finally got to the bottom of this one! It was due to a discrepancy in how extant-ness was inferred. LoadFromFile tries to infer extant-ness entirely from destruction time. If there's no destruction time it just gives up (and optionally assumes all leaves are extant). However, it could potentially also receive information on the number of orgs currently alive. If that number is non-zero, that is *also* an indication that this is an extant taxon. get_mean_pairwise_distance was going entirely off of number of orgs. I have changed LoadFromFile to assume a taxon is extant if its destruction time has not passed OR it has a non-zero number of organisms alive.
